templates/partials/contacts_content.html

```diff
@@ -78,31 +83,51 @@ <h3 class="desktop-only">
       </div>
 
       <div ng-if="selected.children.persons">
-        <h4 translate>People</h4>
+        <div>
+          <div class="col-sm-6"><i class="fa fa-user"></i></div>
+          <div class="col-sm-3 hidden-xs">Title</div>
+          <div class="col-sm-3 hidden-xs">Phone number</div>
+        </div>
         <div class="card action-list children">
           <div class="accent"></div>
           <div class="row" ng-repeat="child in selected.children.persons">
             <div class="col-sm-12">
               <a ui-sref="contacts.detail({ id: child.doc._id })" class="cell">
-                <div class="pull-right" ng-bind-html="child.doc.date_of_birth | age"></div>
-                <i class="fa fa-star primary" ng-show="child.doc._id === selected.doc.contact._id" translate translate-attr-title="Primary contact"></i>
-                {{child.doc.name}}
+                <div>
+                  <div class="col-sm-6">
+                    <i class="fa fa-star primary" ng-show="child.doc.isPrimaryContact" translate translate-attr-title="Primary contact"></i>
+                    {{child.doc.name}}
+                  </div>
+                  <div class="col-sm-3" ng-bind-html="child.doc.title"></div>
+                  <div class="col-sm-3" ng-bind-html="child.doc.phone"></div>
+                </div>
+                <div>
+                  <div ng-show="child.doc.isPrimaryContact">Primary Contact</div>
+                </div>
               </a>
             </div>
           </div>
         </div>
       </div>
 
       <div ng-if="selected.children.places">
-        <h4 translate>{{selected.doc.type + '.field.children'}}</h4>
+        <h4>
+          <i class="fa {{selected.children.childPlacesIcon}}"></i>
+          <span translate>{{selected.children.childPlacesLabel}}</span>
+        </h4>
         <div class="card action-list children">
           <div class="accent"></div>
           <div class="row" ng-repeat="child in selected.children.places">
             <div class="col-sm-12">
               <a ui-sref="contacts.detail({ id: child.doc._id })" class="cell">
-                <div class="pull-right" ng-bind-html="child.doc.date_of_birth | age"></div>
-                <i class="fa fa-star primary" ng-show="child.doc._id === selected.doc.contact._id" translate translate-attr-title="Primary contact"></i>
-                {{child.doc.name}}
+                <div>
+                  <div>
+                    {{child.doc.name}}
+                  </div>
+                </div>
+                <div>
+                  <div ng-show="child.doc.contact.name">Primary Contact: {{child.doc.contact.name}}</div>
```

Comment:
It'll eventually be styled differently, probably, but precise styling is not for right now.